projects/packages/my-jetpack/src/products/class-product.php

```diff
@@ -514,7 +514,7 @@ public static function get_paid_bundles_that_include_product() {
 			return array();
 		}
 		$paid_bundles   = $features['available']->$idendifying_feature ?? array();
-		$current_bundle = Wpcom_Products::get_site_current_plan();
+		$current_bundle = Wpcom_Products::get_site_current_plan( true );
```

Comment:
The only other suggestion I'd have here would be adding a transient cache for about 5 minutes so that we are not re-fetching the plan every time My Jetpack loads.

This is not a blocker for this PR.

Even better would be to cache the result for a long time and invalidate the cache when we detect a change in the site's purchase data - which we already fetch for My Jetpack. But that's better handled in a follow-up task.

Reply:
Personally, I'd avoid merging this PR without at least some level of caching, just to avoid what Josh said about re-fetching every time. I don't oppose to being in a different PR as long as this is included in the same release.